BUILD.bazel

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DISCLAIMER: Bazel support is community-based. The maintainers do not
+# use Bazel internally. The Bazel build can have security risks or
+# optimization gaps.
+
+package(
+    default_visibility = ["//visibility:public"],
+)
+
+cc_library(
+    name = "tbb",
+    srcs = glob([
+        "include/oneapi/tbb/*.h",
+        "include/oneapi/tbb/detail/*.h",
+        "include/tbb/*.h",
+        "src/tbb/*.cpp",
+        "src/tbb/*.h",
+    ]) + [
+        "include/oneapi/tbb.h",
+    ] + select({
+        "@platforms//cpu:x86_64": glob(["src/tbb/tools_api/**/*.h"]),
+        "//conditions:default": [],
+    }),
+    hdrs = glob([
+        "include/tbb/*.h",
+    ]) + select({
+        "@platforms//cpu:x86_64": [
+            "src/tbb/tools_api/ittnotify_static.c",
+        ],
+        "//conditions:default": [],
+    }),
+    copts = [
+        "-w",
+    ],
+    defines =
+        select({
+            "@platforms//cpu:x86_64": ["__TBB_NO_IMPLICIT_LINKAGE"],
+            "//conditions:default": [
+                "USE_PTHREAD",
+            ],
+        }),
+    includes = [
+        "include",
+    ],
+    linkopts =
+        select({
+            "@bazel_tools//platforms:windows": [],
+            "//conditions:default": [
+                "-ldl",
+                "-pthread",
+                "-lrt",
+            ],
+        }),
+    local_defines = select({
+        "@platforms//cpu:x86_64": [
+            "__TBB_USE_ITT_NOTIFY",
+        ],
+        "//conditions:default": [],
+    }) + [
+        "__TBB_BUILD",
+    ],
+)
+
+cc_library(
+    name = "tbbmalloc",
```

Comment:
Speaking about `tbbmalloc`, `tbbmalloc_proxy` and `tbbbind` - these targets are optional, but some functionality/performance may be limited in case of absence of these libraries near the `tbb`. I can't comprehensively list the limitations, because they are quite tricky and sometimes applicable for corner cases.

While `tbbmalloc` and `tbbmalloc_proxy` are in place, I'm not insisting on adding `tbbbind`, I just want to warn you about impact - absence of `tbbbind` may limit some numa-related functionality.

If you decide to add `tbbbind` you'll have to resolve a dependency on hwloc.